src/components/ListItem.jsx

```diff
@@ -86,22 +86,33 @@ export function ListItem({ listPath, item, name }) {
 
 	return (
 		<div className="ListItem">
-			<li>
-				<input
-					checked={isChecked}
-					onChange={handleChange}
-					type="checkbox"
-					id={item.id}
-				/>
-
-				<label className="ListItem-label" htmlFor={item.id}>
-					<p>{name}</p>
+			<li className="ListItem-container">
+				<div className="ListItem-row">
+					<input
+						checked={isChecked}
+						onChange={handleChange}
+						type="checkbox"
+						id={item.id}
+					/>
+					<label className="ListItem-label" htmlFor={item.id}>
+						<p>{name}</p>
+					</label>
+					<button
+						className="delete-btn"
+						aria-label={`Delete ${name}`}
+						onClick={handleDelete}
+					>
```

Comment:
This looks great! I love how easy it is on the eyes. Great job!

Reply:
Thank you!